chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -460,64 +356,186 @@ def process_mmap(self):
         # make lines shorter for style check.
         self._memobj = bitwise.parse(self.MEM_FORMAT, self._mmap)
 
-    def get_settings(self):
-        """Translate the bit in the mem_struct into settings in the UI"""
-        _mem = self._memobj
-        basic = RadioSettingGroup("basic", "Basic Settings")
-        bank = RadioSettingGroup("bank", "Bank names")
-        dtmfe = RadioSettingGroup("dtmfe", "DTMF Encode Settings")
-        top = RadioSettings(basic, bank, dtmfe)
+    def _download(radio):
```

Comment:
This should be `self` now that it's an instance method. It still works because the name is not actually special, but it's very unconventional to have this be anything else and will confuse people into thinking these are a `@staticmethod`.